doc/page-annotations.h

```diff
@@ -39,6 +39,7 @@ The following annotations can be used in structure definitions:
 
 |Annotation                     |              Description                                    | Supported over IPC |
 |-------------------------------|-------------------------------------------------------------|--------------------|
+|\@treatAsByteArray: true       | Treat the whole structure as byte array over IPC reducing structure signature only to DBUS Type ARRAY OF BYTE | Yes             |
```

Comment:
the annotation name should contain "ipc", or "dbus". Something like:
@dbus-serialization: byteArray

Reply:
i think we only need a true/false value, the "byteArray" as a value suggests there are other possibilities as well.